admin/bin/qserv-data-loader.py

```diff
@@ -146,8 +144,10 @@ def __init__(self):
                            help='Disable CSS updates.')
 
         group = parser.add_argument_group('Database options', 'Options for database connection')
-        group.add_argument('-H', '--host', dest='mysqlHost', default='localhost', metavar='HOST',
-                           help='Host name for czar mysql server, def: %(default)s.')
+        group.add_argument('-H', '--host', dest='czarHost', default='localhost', metavar='HOST',
+                           help='Host name for czar wmgr service, def: %(default)s.')
```

Comment:
No, for us there is nothing else in the world but qserv :) Do you have a better name than {{wmgr}}?